README.md

```diff
@@ -36,15 +36,31 @@ Installation
         )
     ```
 
-3. Include the URLs.
+3. Add the required context processor to your list of context processors.
+    ```python
+        TEMPLATES = [
+            {
+                'OPTIONS': {
+                    'context_processors': [
+                        'django.core.context_processors.request',
+                    ],
+                },
+            },
+        ]
+    ```
+
+4. Include the URLs.
     ```python
         urlpatterns = [
             url(r'^admin/', include(admin.site.urls)),
             url(r'^tfc/', include('accession.urls')),
         ]
     ```
+    You may also add the csv export, duplicates, and print urls from the app to
```

Comment:
ah, left a straggler behind. nope, there should be no mention of duplicates anywhere. I'll get rid of it in this comment